app/Packages/TaskFighter/TaskFighterServiceProvider.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TaskFighter;
+
+// Framework
+use Illuminate\Support\Facades\Route;
+use Illuminate\Foundation\Support\Providers\RouteServiceProvider as ServiceProvider;
+
+class TaskFighterServiceProvider extends ServiceProvider
+{
+    /**
+     * The namespace is applied to your controller routes.
+     *
+     * In addition, it is set as the URL generator's root namespace for building
+     * named routes.
+     *
+     * @var string
+     */
+    protected $namespace = 'TaskFighter\Controllers';
+
+    /**
+     * Register services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        //
+    }
+
+    /**
+     * Bootstrap services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        //
+    }
```

Comment:
This is a problem, I think you need to call parent or comment out the stub method.